tests/release/service/tenant_pipelines.go

```diff
@@ -74,9 +74,9 @@ var _ = framework.ReleaseServiceSuiteDescribe("Release service tenant pipeline",
 		tenantPipeline.PipelineRef = tektonutils.PipelineRef{
 			Resolver: "git",
 			Params: []tektonutils.Param{
-				{Name: "url", Value: "https://github.com/jinqi7/integration-examples"},
+				{Name: "url", Value: "https://github.com/redhat-appstudio-qe/pipeline_examples"},
 				{Name: "revision", Value: "main"},
-				{Name: "pathInRepo", Value: "pipelines/integration_pipeline_pass.yaml"},
+				{Name: "pathInRepo", Value: "pipelines/simple_pipeline.yaml"},
```

Comment:
I suggest changing the name of the file in the repo to `simple_tenant_pipeline` instead. Soon we will have the final pipeline as well, and I imagine you might use the same repo for it as well.

Reply:
I actually think this is better. Yes, we will have a final pipeline soon. However, I think e2e should use the same, extremely simple, pipeline for both the final and tenant pipeline test

Reply:
I thought eventually the final pipeline would need to look somehow different.  Fine by me as well, then.